get-details/xdpos2.0-migration.md

```diff
@@ -0,0 +1,62 @@
+# XDPoS 2.0 Migration Steps
+
+- [To Master Owner](#to-master-owner)
+- [To Protocol Team and Private Network Owner](#to-protocol-team-and-private-network-owner)
+- [Verification](#verification)
+
+## To Master Owner
+Upgrading the consensus engine from XDPoS1.0 to XDC2.0 only takes a 3 simple steps.
+
+1. All master node owners confirm their participation by deploying the latest code version.
```

Comment:
commit to the upgrade by deploying the latest code version.

Hi Liam, what is the latest version? It should be a special version that have a super large V2 switch height, right?